tess_area.cpp

```diff
@@ -40,6 +40,10 @@ int main(int argc, char** argv) {
 // only store the positive vertice id numbering
   
   while (inR.read_row(radi)) radius.emplace_back(radi);
+
+// find the maximum radius in order to ignore the left column and bottom layer when plotting postscript.
+  double max_radius = *(std::max_element(radius.begin(), radius.end()));
+
   while (inG.read_row(grain_x,grain_y))grain_xy.emplace_back(std::make_pair(grain_x, grain_y)) ;
```

Comment:
What is this pair supposed to do?

Reply:
This is because I want to ignore the bottom and left layer of the grains by sifting  the grains whose x and y position is greater than the maximum radius but now I don't need them. Thank you for pointing it out!